Library/Homebrew/extend/kernel.rb

```diff
@@ -449,6 +449,8 @@ def redirect_stdout(file)
   }
   def ensure_formula_installed!(formula_name, reason: "", latest: false,
                                 output_to_stderr: true, quiet: false)
+    odeprecated "ensure_formula_installed!", "Formula[\"#{formula_name}\"].ensure_installed!"
```

Comment:
@carlocab @Bo98 This is not a public nor internal API so doesn't need a deprecation cycle. We should just remove it.